configs/crowbar.conf

```diff
@@ -9,8 +9,7 @@
     ErrorDocument 422 /422.html
     ErrorDocument 500 /500.html
 
-    ProxyPass / http://127.0.0.1:3000/ timeout=600 Keepalive=On
+    ProxyPass / http://127.0.0.1:3000/ timeout=1500 Keepalive=On
```

Comment:
We need to do it without proxy_timeout because we have no chef running at this time. It's just for the time while installing the package and start install-chef-suse.